### DISCUSSION TITLE
Не работают войсы в Discord

### DISCUSSION THREAD

**Discussion by finiks12324** [bol-van/zapret#978] [Q&A]
Всем привет, у меня Ubuntu 22.04.5 LTS, и у меня не работают войсы в дискорде (то есть постоянно грузят на подключении к RTC, но дискорд загружается)
Перечитал я все такие же иссуесы, всё перепробовал, ничего не работает (копирование 50-discord в init.d/sysv/custom.d, включение autohostlist в конфиге, изменение конфига и тд)
P.S. я только недавно начал пользоваться Линуксом, поэтому не ругайте плиз

![image](https://github.com/user-attachments/assets/982d8337-3848-4d7e-a17a-bd0c6b436150)

А вот мой конфиг:

https://pastebin.com/79euPFTn

И файлик 50-discord:

https://pastebin.com/kpj9v7zs

**Comment by superpizza2003top** [comment]
Тоже перестали работать войсы, хотя в сам дискорд заходить могу

**Reply by Frestein** [reply]
Ростелеком. Уже недели 2, как минимум, не работают войсы в дискорде.

**Comment by Frestein** [comment]
Целиком конфиг не изучал.
Очевидный треш написанный копипастом.
```
NFQWS_PORTS_UDP=443,50000-50099
NFQWS_OPT="
--filter-tcp=80 --dpi-desync=fake,multisplit --dpi-desync-split-pos=method+2 --dpi-desync-fooling=md5sig <HOSTLIST_AUTO> --new --filter-udp=50000-50099 --dpi-desync=fake --dpi-desync-any-protocol --dpi-desync-fake-quic=0xC3
--filter-tcp=443 --dpi-desync=fake,disorder2 --dpi-desync-split-pos=1 --dpi-desync-ttl=1
--filter-udp=443 --dpi-desync=fake --dpi-desync-repeats=6 <HOSTLIST_NOAUTO>
"
```
Убрал мусор. Это не решит твою проблему, но хотя бы конфигурация будет рабочей.
Остальные имеющиеся косяки (если они имеются) только болван или кто-то также хорошо разбирающийся увидит.
```
NFQWS_PORTS_UDP=443
NFQWS_OPT="
--filter-tcp=80 --dpi-desync=fake,multisplit --dpi-desync-split-pos=method+2 --dpi-desync-fooling=md5sig <HOSTLIST> --new
--filter-tcp=443 --dpi-desync=fake,disorder2 --dpi-desync-split-pos=1 --dpi-desync-ttl=1 <HOSTLIST> --new
--filter-udp=443 --dpi-desync=fake --dpi-desync-repeats=6 <HOSTLIST_NOAUTO>
"
```

У тебя в принципе ни nfqws, ни tpws не включены. Мод фильтр тоже не выбран. Не знаю какой работы ты ожидал. RTFM

**Reply by finiks12324** [reply]
А как выбрать? Сори, я слишком глупый

**Reply by Frestein** [reply]
> А как выбрать? Сори, я слишком глупый

Ознакомься:
https://github.com/bol-van/zapret/blob/master/docs/quick_start.md
